tests/functional/api/test_query.py

```diff
@@ -24,6 +25,25 @@ def test_basic_query(eth_tester_provider):
     ]
 
 
+def test_block_transaction_query(sender, receiver, chain, eth_tester_provider):
+    eth_tester_provider.get_transactions_by_block("0")
```

Comment:
Can we make assertions about the result from this call as well?

Reply:
I removed this, we didn't need it